docs/services/navigation/_index.md

```diff
@@ -554,3 +554,137 @@ obstacles = await my_nav.get_obstacles()
+The movement sensor API's [`GetOrientation()`](/components/movement-sensor/#getorientation) takes orientation readings.
+Additionally, follow [these instructions](/services/frame-system/#configuration) to configure the geometries of each component of your robot within the [frame system](/services/frame-system/).
+Use orientation readings to determine the orientation of an object in 3D space as an "orientation vector", or, its position within the [cartesian coordinate system](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) relative to some specific `origin` point that you, the user, need to choose and configure for your robot in Viam's [frame system](/services/frame-system/).
+
+### Angular Velocity
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of the [movement sensor](/components/movement-sensor/) component take angular velocity measurements:
+
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+- [imu-vectornav](/components/movement-sensor/imu/imu-vectornav/)
+- [wheeled-odometry](/components/movement-sensor/wheeled-odometry/)
+- [gyro-mpu6050](/components/movement-sensor/mpu6050/)
+
+An example of an `AngularVelocity` reading:
+
+``` go
+// angularVelocity is an AngularVelocity{X:1, Y:2, Z:3}
+angularVelocity, err := imu.AngularVelocity{context.Background, nil}
+```
+
+If you want to get an angular velocity reading, first [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+The movement sensor API's [`GetAngularVelocity()`](/components/movement-sensor/#getangularvelocity) takes angular velocity readings.
+Use angular velocity readings to determine the speed and direction at which your robot is rotating.
+
+### Position
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of the [movement sensor](/components/movement-sensor/) component take position measurements:
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+
+An example of a `Position` reading:
+
+``` go
+// position is a geo.Point{Lat: 40.7, Long: -73.98}, altitude is a float64 100
+position, altitude, err:= imu.Position{context.Background, nil}
+```
+
+If you want to get a position, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+The movement sensor API's [`GetPosition()`](/components/movement-sensor/#getposition) takes position readings.
+The suggested components allow you to get the *absolute* position of components through their Positions readings for use in the [motion service](/services/motion/) and [navigation service](/services/navigation/).
+Use position readings to determine the GPS coordinates of an object in 3D space or its position in the geographic coordinate system [(GCS)](https://en.wikipedia.org/wiki/Geographic_coordinate_system).
```

Comment:
```suggestion
Use position readings to determine the GPS coordinates of an object in 3D space or its position in the geographic coordinate system [(GCS)](https://en.wikipedia.org/wiki/Geographic_coordinate_system).
These position readings reflect the *absolute* position of components.

To get a position, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
Then use the movement sensor API's [`GetPosition()`](/components/movement-sensor/#getposition) method to get position readings from the sensor.
```
Same as above; I think what and why should come before how.

Reply:
Sounds good to me, done